config/localization.yaml

```diff
@@ -10,6 +10,10 @@ default:
             successfulOrder: Thank you for shopping with us!
             successfulLogin: Login successful. Have fun in our shop!
             successfulAccountDeletion: Your account has been deleted. We hope to see you soon again!
+        posterSlider:
+            title: Check out our new panorama posters
+            numSlides: 4
```

Comment:
numSlides and activeSlideClass should not be localized, keep them in the java world